llm/emergency_parser.py

```
@@ -0,0 +1,122 @@
+        for part in parts:
+            part = part.strip()
+            if part.startswith("{") and part.endswith("}"):
+                return part
+
+    start = text.find("{")
+    end = text.rfind("}")
+
+    if start != -1 and end != -1:
+        return text[start:end + 1]
+
+    raise ValueError("JSON 영역을 찾을 수 없습니다.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)